### PR TITLE
Release version 0.6.0

### PR DESCRIPTION
**Pull request by comit-botty-mc-botface**
Hi @da-kami!

This PR was created in response to a manual trigger of the release workflow here: https://github.com/comit-network/xmr-btc-swap/actions/runs/870083908.
I've updated the changelog and bumped the versions in the manifest files in this commit: 30b2cb467a8dde873da16cedf69f5b4ccbdb7508.

Merging this PR will create a GitHub release and upload any assets that are created as part of the release build.

### CODE REVIEW

**Comment by rishflab** [comment]
Im confused here, is this a fix up PR? there doesn't seem to be any code changes apart from an upgrade to 0.6.0?

**Comment by da-kami** [comment]
> Im confused here, is this a fix up PR? there doesn't seem to be any code changes apart from an upgrade to 0.6.0?

This is the PR created by the prepare release action. There are no code changes, besides the Changelog update and the version update. All code changes will be listed under the newly created section `[0.6.0] - 2021-05-24` in the changelog. 

Once this PR is merged the release action will pick it up and do the actual release.

**Comment by da-kami** [comment]
Note: I gave the mac preview release binaries a quick go on my machine, all good.

**Comment by da-kami** [comment]
One thing I noticed that might become a problem: 

We currently configure the default `mainnet` public node using `node.moneroworld.com:18089` as specified on https://moneroworld.com/#nodes

I ran several balance checks using he ASB and noticed that I do get errors in the monero-wallet-rpc:

```
2021-05-24 04:23:54.852	E !r. THROW EXCEPTION: tools::error::no_connection_to_daemon
2021-05-24 04:23:54.857	E Exception at while refreshing, what=no connection to daemon
```

When using `node.xmr.to:18081` I did not encounter such errors.

It might be that `node.moneroworld.com:18089` is not stably running - unfortunately don't have too much experience with that. 
Should we change the default node? @thomaseizinger @rishflab

**Comment by da-kami** [comment]
> One thing I noticed that might become a problem:
> 
> We currently configure the default `mainnet` public node using `node.moneroworld.com:18089` as specified on https://moneroworld.com/#nodes
> 
> I ran several mainnet balance checks using he ASB and noticed that I do get errors in the monero-wallet-rpc:
> 
> ```
> 2021-05-24 04:23:54.852	E !r. THROW EXCEPTION: tools::error::no_connection_to_daemon
> 2021-05-24 04:23:54.857	E Exception at while refreshing, what=no connection to daemon
> ```
> 
> When using `node.xmr.to:18081` I did not encounter such errors.
> 
> It might be that `node.moneroworld.com:18089` is not stably running - unfortunately don't have too much experience with that.
> Should we change the default node? @thomaseizinger @rishflab

I decided that we better not take changes and changed the default. Also updated the ASB readme.

File: docs/asb/README.md [hunk 1]
```diff
@@ -21,11 +21,12 @@ Run `./asb --help` for more information.
...
 ### Running on mainnet
 
-Public Monero mainnet nodes can be found [here](https://moneroworld.com/#nodes).
-The default mainnet node is set to `node.moneroworld.com:18089`.
-
 For running the ASB on mainnet you will have to change you `monero-wallet-rpc` setup to mainnet.
-The default port for expecting the `monero-wallet-rpc` when running the ASB on mainnet is `18083`.
+
+It is recommended that you run your own Monero and Bitcoin node when running on mainnet.
+It is possible to plug into publicly blockchain nodes but be aware that you might lose some privacy doing so.
```
Comment (thomaseizinger) [inline docs/asb/README.md:27]: Typo: `publicly`

**Review by thomaseizinger — APPROVED** [review]
Nice!

**Review by rishflab — APPROVED** [review]

**Comment by da-kami** [comment]
Note: ran into a Github upload issue upon artifact upload, re-running.

**Pull request merged**